File: 3085.java
import java.util.Scanner;


public class Main {
 public static void main(String[] args)
 {
	Scanner in = new Scanner(System.in);
	int n = in.nextInt();
	for(int i = 0; i < n; i++)
	{
	int sum = in.nextInt();
	int quarter = sum/25;
	sum = sum %25;
	int dime = sum/10;
	sum = sum%10;
	int nickel = sum/5;
	sum = sum%5;
	int penny = sum;
	System.out.println((i+1)+" "+quarter+" QUARTER(S), "+dime+" DIME(S), "+nickel+
		" NICKEL(S), "+penny+" PENNY(S)");
	}
  }
}

File: 3085.cc
#include <iostream>
using namespace std;

int main() {
  int t;
  cin >> t;
  int x;

  int x1, x2, x3, x4;
  for (int i = 0; i < t; i++) {
    cin >> x;
    x1 = x / 25;
    x -= x1 * 25;
    x2 = x / 10;
    x -= x2 * 10;
    x3 = x / 5;
    x -= x3 * 5;
    x4 = x / 1;

    cout << i + 1 << " " << x1 << " QUARTER(S), " << x2 << " DIME(S), " << x3
         << " NICKEL(S), " << x4 << " PENNY(S)" << endl;
  }
}
